bindings/rascal/utils/filter_utils.py

```diff
@@ -62,7 +62,6 @@ def get_index_mappings_sample_per_species(managers, sps):
 
     return strides_by_sp, global_counter, map_by_manager, indices_by_sp
 
-
```

Comment:
Is this auto-formatting? PEP8 says two blank lines between top-level functions, so I'd just keep it that way.

Reply:
done